source/03-components/call-to-action/call-to-action.twig

```diff
@@ -0,0 +1,28 @@
+{% set classes = [
+  'c-call-to-action',
+  modifier_classes ? modifier_classes : ''
+] %}
+
+{% set cta_content %}
+  <div class="c-call-to-action__media">
+    {{ media }}
+  </div>
+  <div class="c-call-to-action__copy">
+    {% if heading %}
+      <h2>{{ heading }}</h2>
+    {% endif %}
+    {{ body }}
+    {% if link.url and link.text %}
+      <div>
+        <a href="{{ link.url }}" class="c-button">{{ link.text }}</a>
+      </div>
+    {% endif %}
```

Comment:
@jbwagner in your yaml, these are non-nested variables called 'title' and 'url'

Reply:
That was intended to be non-nested, something I changed when testing inside a Drupal install but seem to have missed when making those changes on this repo. [a16c5d9](https://github.com/forumone/gesso/pull/886/commits/a16c5d9c8f28b562529738131b4f0150d33fc758)